Plugin~/Src/MeshSync/Include/MeshSync/SceneGraph/msEntityConverter.h

```diff
@@ -13,6 +13,7 @@ class Points;
 class Animation;
```

Comment:
Can't find a good place to write a comment about the DLL, so I am putting it here.

I remember that you tried to set up a Linux virtual machine, have you done that ?
If not, I'd suggest you to set it up, so that we can have the Linux so file update together.

If you have a Mac, I'd also like to ask you to set it up similarly, but if not, then I guess I would be the one who have to update it. 
May be better to have someone who have a Mac in your timezone do this though.
Perhaps @seandillon92 can help ?

Reply:
I think we need to assign that to someone then. Feel free to take it !
We can also discuss about this in our next meeting.